test/donations.routes.test.js

```diff
@@ -29,7 +30,7 @@ const stripeMock = require('./mocks/stripe');
 /**
  * Tests.
  */
-describe('payments.routes.test.js', () => {
+describe('donations.routes.test.js', () => {
```

Comment:
Not sure why this test still calls `.post('/groups/' + group.id + '/payments')` (line 206) which is supposed to be a deprecated route cfr. comment https://github.com/OpenCollective/api/pull/234/files#diff-6db976b763ed71154781e45807c0d4e5R127